getting_started.rst

```diff
@@ -1,170 +1,160 @@
 .. _getting_started:
 
+
 Getting started
 ===============
 
-As an example, let's start with one of the most popular C++ libraries: POCO_.
-Conan **works with any build system** and it does not depend on CMake, though we will use CMake for
-this example for convenience.
+Let's start with an example using one of the most popular C++ libraries: POCO_. For convenience purposes we'll use CMake.
+Keep in mind that Conan **works with any build system** and does not depend on CMake.
 
 .. _POCO: https://pocoproject.org/
 
+
 A Timer using POCO libraries
 ----------------------------
 
 First, let's create a folder for our project:
 
 .. code-block:: bash
 
-    $ mkdir mytimer
-    $ cd mytimer
-
+   $ mkdir mytimer
+   $ cd mytimer
+   
 .. note::
 
-    If you have the code in a github repository, instead of creating the folder, you can just clone
-    the project:
-
+    If your code is in a GitHub repository you can simply clone the project, instead of creating this folder, using the following command:
+    
     .. code-block:: bash
     
        $ git clone https://github.com/memsharded/example-poco-timer.git mytimer
+       
+       
+Next, create the following source files inside this folder:
+
 
-Create the following source files inside that folder:
+**timer.cpp**
```

Comment:
Leave the file name as :caption: as it was originally.
Also, as above, do not reindent the source code inside code-blocks.